webapp/src/components/onboardingTour/tourTipRenderer/tourTipRenderer.tsx

```diff
@@ -33,19 +31,17 @@ type Props = {
 
 const TourTipRenderer = (props: Props): JSX.Element | null => {
     const board = useAppSelector(getCurrentBoard)
-    const clientConfig = useAppSelector<ClientConfig>(getClientConfig)
 
     let isOnboardingBoard = board ? board.title === OnboardingBoardTitle : false
     const onboardingTourStarted = useAppSelector(getOnboardingTourStarted)
     const onboardingTourCategory = useAppSelector(getOnboardingTourCategory)
     const onboardingTourStep = useAppSelector(getOnboardingTourStep)
-    const disableTour = clientConfig?.featureFlags?.disableTour || false
 
     if (props.showForce) {
         isOnboardingBoard = true
     }
 
-    const showTour = !disableTour && isOnboardingBoard && onboardingTourStarted && onboardingTourCategory === props.category
```

Comment:
We do want to keep checking this setting given that this will not show the onboarding tour in Boards if it was disabled in the server

Reply:
Okay will be reverting this code back also for fetching `FeatureFlag` from server, but as of now we are not getting any flag from the server with the name `disableTour`.